bot/cogs/bot.py

```diff
@@ -153,28 +193,41 @@ def codeblock_stripping(self, msg: str):
                         # This check is to avoid all nodes being parsed as expressions.
                         # (e.g. words over multiple lines)
                         if not all(isinstance(node, ast.Expr) for node in tree.body):
-                            codeblock_tag = await self.bot.get_cog("Tags").get_tag_data("codeblock")
-
-                            if codeblock_tag == {}:
-                                log.warning(f"{msg.author} posted something that needed to be put inside Python "
-                                            "code blocks, but the 'codeblock' tag was not in the tags database!")
-                                return
-
-                            log.debug(f"{msg.author} posted something that needed to be put inside python code blocks. "
-                                      "Sending the user some instructions.")
-                            howto = (f"Hey {msg.author.mention}!\n\n"
-                                     "I noticed you were trying to paste code into this channel.\n\n"
-                                     f"{codeblock_tag['tag_content']}")
-
-                            howto_embed = Embed(description=howto)
-                            await msg.channel.send(embed=howto_embed)
-                            self.channel_cooldowns[msg.channel.id] = time.time()
-
-                    except SyntaxError:
-                        log.trace(f"{msg.author} posted in a help channel, and when we tried to parse it as Python "
-                                  f"code, ast.parse raised a SyntaxError. This probably just means it wasn't Python "
-                                  f"code. The message that was posted was:\n\n{msg.content}\n\n")
-                        pass
+                            # Shorten the code to 10 lines and/or 204 characters.
+                            space_left = 204
+                            if len(content) >= space_left:
+                                current_length = 0
+                                lines_walked = 0
+                                for line in content.splitlines(keepends=True):
+                                    if current_length+len(line) > space_left or lines_walked == 10:
+                                        break
+                                    current_length += len(line)
+                                    lines_walked += 1
+                                content = content[:current_length]+"#..."
+
+                                howto += (f"Hey {msg.author.mention}!\n"
```

Comment:
This is indented one level too far, which means nothing happens if the code is shorter than 204 characters.